### PR TITLE
Restructure user guide

### PR DESCRIPTION
**Pull request by wangcj05**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
#162 

##### What are the significant changes in functionality due to this change request?
N/A

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code. *No computer code, but other changes reviewed*
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts). *No changes*
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details. *No code*
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass. *Yes*
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True. *No new functionality*
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync. *No change*
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.  *Done*
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added? *No change*

### CODE REVIEW

**Comment by wangcj05** [comment]
@alfoa Do you have time to take a look at this PR?

**Comment by alfoa** [comment]
i can do that.

**Comment by joshua-cogliati-inl** [comment]
I'll look at it.

File: doc/user_guide/singleRunAndPlot.tex [hunk 1]
```diff
@@ -1,135 +1,265 @@
...
+
+For examples of the numerical data produced by the OutStreams \textit{Print}, see \texttt{history\_0.csv} in the directory
+ \texttt{raven/tests/framework/user\_guide/ravenTutorial/ gold/singleRunPlot/}
  As previously mentioned, Figure~\ref{fig:historyPlotLine} reports the four plots (four variables) drawn in the same picture.
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %\FloatBarrier
-\subsection{Single Run using the OutStream System to Sub-plot and Selectively print.}
-This Section shows how to use RAVEN to create sub-plots (multiple plots in the same figure) and
+\subsubsection{Sub-plot and selectively print.}
```
Comment (joshua-cogliati-inl) [inline doc/user_guide/singleRunAndPlot.tex]: "and selectively print." should probably be "and selectively printing."

File: doc/user_guide/exampleDescription.tex [hunk 2]
```diff
@@ -19,21 +16,20 @@ \subsection{AnalyticBateman, Example Model}\label{sec:analyticalbateman}
...
    \end{itemize}
 
-For example, this  code is able to solve a system of equations as follows:
+For example, this  code is able to solve a system of two ODEs as follows:
 \begin{equation}
   \begin{dcases}
-   \frac{\mathrm{d} x_{1}}{\mathrm{d} t} = \phi (t)\times \sigma_{x_{1}}-\lambda_{x_{1}} \\
-   \frac{\mathrm{d} x_{2}}{\mathrm{d} t} = \phi (t)\times \sigma_{x_{2}}-\lambda_{x_{2}}+x_{1}(t)\times\lambda_{x_{1}} \\
+    \frac{\mathrm{d} x_{1}}{\mathrm{d} t} = \phi (t)\times \sigma_{x_{1}}-\lambda_{x_{1}}\times x_{1}(t) \\
+    \frac{\mathrm{d} x_{2}}{\mathrm{d} t} = \phi (t)\times \sigma_{x_{2}}-\lambda_{x_{2}} \times x_{2}(t) + x_{1}(t)\times\lambda_{x_{1}} \\
```
Comment (joshua-cogliati-inl) [inline doc/user_guide/exampleDescription.tex]: Just to double check, changing the equations was intended?

File: doc/user_guide/ravenComponents.tex [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  RAVEN run.
+  %
+  \item \xmlNode{OutStreams}: Visualization and
+  Printing system block.
+  %
+  \item \xmlNode{Models}: Specifies codes, ROMs,
+  post-processing analysis, etc.
+  %
+  \item \xmlNode{Functions}: Details interfaces to external
+  user-defined functions and modules. The user will be building and/or running.
```
Comment (wangcj05) [inline doc/user_guide/ravenComponents.tex]: fixed.

File: doc/user_guide/ravenPostProcess.tex [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+As before, all defined RAVEN entities are combined in the \xmlNode{Steps} block.
+\xmlExample{framework/user_guide/ravenTutorial/PostProcess.xml}{Steps}
+In this case, three steps have been defined:
+
+\begin{itemize}
+  \item \xmlNode{MultiRun} named ``sampleMC'', used to run the multiple instances of the driven code and
+     collect the outputs in the two \textit{DataObjects}. As it can be seen, the \xmlNode{Sampler} is inputted
+     to communicate to the \textit{Step} that the driven code needs to be perturbed through the Monte Carlo sampling
+     strategy.
+  \item \xmlNode{PostProcess} named ``statAnalysis\_1'', used compute all the statistical moments and FOMs
```
Comment (wangcj05) [inline doc/user_guide/ravenPostProcess.tex]: you are right, fixed.

File: doc/user_guide/singleRunAndPlot.tex (hunk 1 above)
Comment (wangcj05) [inline doc/user_guide/singleRunAndPlot.tex]: fixed.

File: doc/user_guide/exampleDescription.tex (hunk 2 above)
Comment (wangcj05) [inline doc/user_guide/exampleDescription.tex]: Yes, the original equations do not look right to me.

**Pull request merged**